Source/WebCore/page/LocalFrame.h

```diff
@@ -170,6 +170,7 @@ class LocalFrame final : public Frame {
     void resetScript();
 
     WEBCORE_EXPORT bool isRootFrame() const final;
+    WEBCORE_EXPORT LocalFrame& rootFrame();
```

Comment:
This would be clearer as `rootmostLocalFrame()`

Reply:
I chose rootFrame to be more similar to Frame::mainFrame.  All root frames are LocalFrames, so rootmostLocalFrame would be redundant naming.

Reply:
"All root frames are LocalFrames" — that may be clear to you, but it's not to me.